python/omnifold.py

```diff
@@ -212,16 +212,13 @@ def omnifold(
 
     return weights_unfold
 
-# evaluate a step in OmniFold method, training a model if needed and train the model
-# weights_push is modified for reweighting
-# Return: None
 def train_step(
```

Comment:
Sorry for asking again about the function name, but with the updates, maybe `reweight_step` would be a more suitable name?

Reply:
Yes you are right. The original plan was to handle all training and weight updating in one function. Now the main purpose of the function is to train the model and return the updated weights. How about get_step_reweight?